Hello.java

```diff
@@ -0,0 +1,9 @@
+public class Hello {
+
+       Hello(){
+           System.out.print("Hello, World!");
+       }
+       public static void main(String[] args) {
+            Hello helloWorld = new Hello();
```

Comment:
The role of the constructor is to construct the object for subsequent use as allowed by it's methods. Here, the constructor does all the useful work itself. So the client (main) has no use for the object after it's constructed.